bundles/org.openhab.binding.rotel/src/main/java/org/openhab/binding/rotel/internal/handler/RotelHandler.java

```diff
@@ -2349,76 +2181,430 @@ private void updateChannelState(String channel) {
     }
 
     /**
-     * Inform about the main zone power state
+     * Inform about the device / main zone power state
      *
-     * @return true if main zone power state is known and known as ON
+     * @return true if device / main zone power state is known and known as ON
      */
     private boolean isPowerOn() {
-        Boolean power = this.power;
-        return power != null && power.booleanValue();
+        return isPowerOn(0);
+    }
+
+    /**
+     * Inform about the power state
+     *
+     * @param numZone the zone number (1-4) or 0 for the device or main zone
+     *
+     * @return true if power state is known and known as ON
+     */
+    private boolean isPowerOn(int numZone) {
+        Boolean power = powers[numZone];
+        return (numZone > 0 && !powerControlPerZone) ? isPowerOn(0) : power != null && power.booleanValue();
     }
 
     /**
-     * Get the command to be used for main zone POWER ON
+     * Get the command to be used for POWER ON
+     *
+     * @param numZone the zone number (2-4) or 0 for the device or main zone
      *
      * @return the command
      */
-    private RotelCommand getPowerOnCommand() {
-        return model.hasOtherThanPrimaryCommands() ? RotelCommand.MAIN_ZONE_POWER_ON : RotelCommand.POWER_ON;
+    private RotelCommand getPowerOnCommand(int numZone) {
+        switch (numZone) {
+            case 0:
+                return model.hasOtherThanPrimaryCommands() ? RotelCommand.MAIN_ZONE_POWER_ON : RotelCommand.POWER_ON;
+            case 2:
+                return RotelCommand.ZONE2_POWER_ON;
+            case 3:
+                return RotelCommand.ZONE3_POWER_ON;
+            case 4:
+                return RotelCommand.ZONE4_POWER_ON;
+            default:
+                throw new IllegalArgumentException("No power ON command defined for zone " + numZone);
+        }
     }
 
     /**
-     * Get the command to be used for main zone POWER OFF
+     * Get the command to be used for POWER OFF
+     *
+     * @param numZone the zone number (2-4) or 0 for the device or main zone
      *
      * @return the command
      */
-    private RotelCommand getPowerOffCommand() {
-        return model.hasOtherThanPrimaryCommands() ? RotelCommand.MAIN_ZONE_POWER_OFF : RotelCommand.POWER_OFF;
+    private RotelCommand getPowerOffCommand(int numZone) {
+        switch (numZone) {
+            case 1:
```

Comment:
Not sure, but if it should work as documented and aligned with `getPowerOnCommand`:
```suggestion
            case 0:
```

Reply:
Oups, good catch

Reply:
Fixed